src/compiler/types.ts

```diff
@@ -999,6 +999,7 @@ namespace ts {
     export interface StringLiteral extends LiteralExpression {
         kind: SyntaxKind.StringLiteral;
         /* @internal */ textSourceNode?: Identifier | StringLiteral | NumericLiteral; // Allows a StringLiteral to get its text from another node (used by transforms).
+        /* @internal */ singleQuote?: boolean;

```

Comment:
A `StringLiteral` can't be quoted using backtick. If it is quoted using backtick in source, then it would be a `NoSubstitutionTemplateLiteral`